app/cinema/hall.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable, List
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: List[Callable],
+            clearing_staff: Callable
+    ) -> None:
+        print(f'"{movie_name}" started in hall number {self.number}.')
+        for customer in customers:
+            customer.watch_movie(movie_name)
+        print(f'"{movie_name}" ended.')
+        clearing_staff.clean_hall(self.number)
```

Comment:
Consider adding empty lines before and after a loop which can significantly enhance code readability and make the review process easier. This practice helps in visually separating different code blocks, making the structure of the code clearer. 
```suggestion
        print(f'"{movie_name}" started in hall number {self.number}.')
        
        for customer in customers:
            customer.watch_movie(movie_name)
        
        print(f'"{movie_name}" ended.')
        clearing_staff.clean_hall(self.number)
```